### PR TITLE
Replace GOT with AXIOS

### PR DESCRIPTION
**Pull request by neil-stephen-mcgonigle**
Migrating to AXIOS due to audit failure with GOT dependencies.
There are still issues this build uses AXIOSv0.27.2 I would be happier if we could get this working with v1.3.3
We are currently blocked from deploying to UAT
This build removes the blocker

### CODE REVIEW

File: questionnaire/request-service.js [hunk 1]
```diff
@@ -10,12 +10,11 @@ function requestService() {
             headers: {
                 accept: 'application/vnd.api+json',
                 'Content-Type': 'application/vnd.api+json'
-            },
-            responseType: 'json',
-            throwHttpErrors: false
+            }
+            // validateStatus: () => true //never throw errors
```
Comment (neil-stephen-mcgonigle) [inline questionnaire/request-service.js:14]: Review whether this is necessary to replace GOT's `throwHttpErrors: false`

File: questionnaire/questionnaire-service.js [hunk 2]
```diff
@@ -107,26 +107,26 @@ function questionnaireService() {
...
-            (result.body.errors && result.body.errors[0].status === 404)
+            !result.data ||
+            !result.data.data ||
+            !result.data.data.attributes ||
+            (result.data.errors && result.data.errors[0].status === 404)
         ) {
             const err = Error(`The service is currently unavailable`);
             err.name = 'DCSUnavailable';
-            err.statusCode = 500;
+            err.status = 500;
```
Comment (neil-stephen-mcgonigle) [inline questionnaire/questionnaire-service.js:117]: Do we have a test in place for this, is it a valid integration test? 
Will need to test this within an integration environment.

File: questionnaire/request-service.js (hunk 1 above)
Comment (neil-stephen-mcgonigle) [reply questionnaire/request-service.js]: It is required otherwise validation error handling will brake.

**Comment by neil-stephen-mcgonigle** [comment]
Proposed version bump patch

**Review by BarryPiccinni — APPROVED** [review]

**Pull request closed without merging**